src/usr/share/batwatch/batwatch.py

```diff
@@ -53,124 +46,150 @@ def __eq__(self, other):
-        """
-        Send an e-mail to the configured gpgmailer recipient when battery state changes.
-
-        current_status: The previous battery CompositeStatus.
-        new_status: The updated battery CompositeStatus.
-        """
-
-        email = gpgmailmessage.GpgMailMessage()
-        # get subject from self.config or else leave blank for gpgmailer default.
-        if (self.config['email_subject']):
-            email.set_subject(self.config['email_subject'])
-
-        body = self._build_email_body(current_status, new_status)
-        email.set_body(body)
+    def _build_email_body(self, prior_status, current_status):
+        """Creates an e-mail body with the battery status change information.
 
-        email.queue_for_sending()
+        prior_status: The previous battery CompositeStatus.
+        current_status: The updated battery CompositeStatus.
 
-    def _build_email_body(self, current_status, new_status):
+        Returns a string intended to be an e-mail body.
         """
-        Return the text with battery status change information to be sent in the e-mail alert.
 
-        current_status: The previous battery CompositeStatus at beginning of this loop.
-        new_status: The updated battery CompositeStatus.
-        returns: E-mail body text including previous and updated battery status details.
-        """
+        body_text = 'Batwatch detected a change in your device\'s battery status:\n\n'
 
-        body_text = "Batwatch detected a change in your device's battery status:\n\n"
-
-        count_diff = new_status.battery_count - current_status.battery_count
+        count_diff = current_status.battery_count - prior_status.battery_count
         if count_diff != 0:
             if count_diff < 0:
-                body_text += 'A battery was ADDED.'
+                body_text += 'Batteries were added.'
             elif count_diff > 0:
-                body_text += 'A battery was REMOVED.'
-            body_text += ' Count of batteries is now {}, was {}.\n\n' \
-                .format(new_status.battery_count, current_status.battery_count)
+                body_text += 'Batteries were removed.'
+
+            body_text += ' Count of batteries is now %s, was %s.\n\n' % (
+                current_status.battery_count, prior_status.battery_count)
 
-        if new_status.charge_status != current_status.charge_status:
-            body_text += 'The battery is now {}. Previously, it was {}.\n\n' \
-                .format(CHARGE_STATUS_DICT[new_status.charge_status], CHARGE_STATUS_DICT[current_status.charge_status])
+        if current_status.charge_status != prior_status.charge_status:
+            body_text += 'The battery is now %s. Previously, it was %s.\n\n' % (
+                CHARGE_STATUS_LIST[current_status.charge_status],
+                CHARGE_STATUS_LIST[prior_status.charge_status])
 
         return body_text
+
+    def _send_email(self, body_text):
+        """ Send an email.
```

Comment:
Remove the leading space. I'd refer to this in the present tense. "Sends an e-mail." Also "e-mail"! 😠